### DISCUSSION TITLE
可否修改“复制链接”功能，加入视频标题？

### DISCUSSION THREAD

**Discussion by huangsijun17** [the1812/Bilibili-Evolved#2557] [功能建议 / Ideas]
在聊天等场景时，单发一个链接，但却不带上标题，略显不雅。建议在复制链接时加上(可选)视频标题以及视频出处是B站。

**Comment by the1812** [comment]
哪个地方的复制链接

**Reply by huangsijun17** [reply]
![image](https://user-images.githubusercontent.com/31504595/143671055-73336ceb-307e-4da8-abcb-761e2022ff92.png)

**Comment by Geeyun-JY3** [comment]
推荐：https://github.com/the1812/Bilibili-Evolved/issues/2294#issuecomment-927976980
https://greasyfork.org/scripts/398655
